README.md

```diff
@@ -20,23 +20,136 @@ Required parameters:
 
 | Parameter        | Type  |  Description |
 | :-------------   | :---- | :-------------|
-| scmUrl        | String | Scm Url to format |
+| scmUrl        | String | Scm url to format |
 
 #### Expected Outcome
 A formatted scm url to be used as a unique key. This function is synchronous.
 
+### parseUrl
+Required parameters:
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| config             | Object | Configuration Object |
+| config.checkoutUrl | String | Checkout url for a repo to parse |
```

Comment:
Needs a `token`. Part of the SCM identifier needs to be determined by the SCM service. This requires a query against the SCM service, which is why we need a `token`